js-exercises/args-string/argsString.js

```diff
@@ -0,0 +1,16 @@
+const argsString = (message, args) => {
+  const messsageList = message.split('{}');
+  let argsIndex = 0;
+  let modifiedMessage = message;
+
+  for (const item of messsageList) {
+    if (item === '' && args[argsIndex]) {
```

Comment:
args[argsIndex] could be stored in a variable because you are using it multiple times.

Reply:
I assigned the value to a variable as suggested. Please https://github.com/pesto-students/deep-dive-ajay14312/pull/3/commits/0e9e9c30c654a68aefc3ec60d6397654275a03b2